merged.pem

```diff
@@ -0,0 +1,61 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this needed?

Reply:
so terraform was complaining if the merged.pem file does not exist in the validation step hence I had to add it to the module

Reply:
maybe just blank file? Could be confusing for someone coming along to amend.

Can you update readme as well?

Reply:
done